server/http.go

```diff
@@ -162,7 +162,7 @@ func (p *Plugin) completeUserOAuth(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	user, appErr := p.API.GetUser(userID)
+	_, appErr := p.API.GetUser(userID)
 	if appErr != nil {
 		p.API.LogError("complete oauth, error getting MM user", "error", appErr.Error())
 		http.Error(w, appErr.Error(), http.StatusInternalServerError)
```

Comment:
Can we remove this function call now?

Reply:
@raghavaggarwal2308 The function call is useful in validating the userID, should we remove it?

Reply:
I dont think there is any need to do that, please remove it